latch_cli/services/register/register.py

```diff
@@ -61,8 +70,10 @@ def print_and_write_build_logs(
     pkg_root: Path,
     *,
     progress_plain: bool = False,
+    print_build_message: bool = True,
 ):
-    click.secho(f"Building Docker image", bold=True)
+    if print_build_message:
```

Comment:
Doesn't seem like you then use this to control actual print of build message

Reply:
its for the "Building Docker image" message only